src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -324,7 +325,9 @@ void Graph::Replicate(const CNNNetwork &network) {
         const auto parentEdges = output.second->getParentEdgesAtPort(0);
         for (size_t i = 0; i < parentEdges.size(); i++) {
             const auto parent = parentEdges[i]->getParent();
-            parent->setOriginalOutputPrecisionAtPort(parentEdges[i]->getInputNum(), precToSet);
+            if (parent->getOriginalOutputPrecisionAtPort(parentEdges[i]->getInputNum()) != Precision::FP16) {
```

Comment:
it seems we need to update normalizeToSupportedPrecision method for fp16 case

Reply:
fixed